docs/turbine_interaction.md

```diff
@@ -0,0 +1,142 @@
+---
```

Comment:
@RHammond2 Is it possible to convert this to a notebook? This would allow for it to be run and interacted with locally as well as be included in the docs.

Reply:
I think that should be easy to do since I didn't do anything Jupyter can't handle in the markdown.

Reply:
It's now a notebook!